modules/apps/export-import/export-import-test-util/src/main/java/com/liferay/exportimport/test/util/lar/BasePortletExportImportTestCase.java

```diff
@@ -390,7 +390,7 @@ protected void exportImportPortlet(
 						TYPE_PUBLISH_PORTLET_LOCAL,
 					settingsMap);
 
-		ExportImportThreadLocal.setPortletStagingInProcess(true);
+		ExportImportThreadLocal.setPortletExportInProcess(true);
```

Comment:
@joseabelenda, this change modifies a base class, probably, used by several other test classes, not only the Forms classes, in light of this should we really change it? After all, it'll probably affect other products. Is there any way to circumvent the test failures only changing Forms classes?